examples/advanced/torque-controls.html

```diff
@@ -0,0 +1,188 @@
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.1/mapbox-gl.css" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,400,500" rel="stylesheet">
+    <style type="text/css">
+        * {
+            margin: 0;
+            padding: 0;
+        }
+
+        body {
+            display: flex;
+            height: 100vh;
+        }
+
+        #sidebar {
+            padding: 25px;
+            width: 275px;
+            background: #f8f8f8;
+        }
+
+        #sidebar h1 {
+            font: 100 42px/1.4 'Roboto';
+            margin-bottom: 10px;
+        }
+
+        #sidebar h2 {
+            font: 200 21px/1.4 'Roboto';
+            margin-bottom: 10px;
+        }
+
+        #sidebar div {
+            font: 300 14px/1.6 'Roboto';
+            margin-bottom: 20px;
+        }
+
+        #sidebar div input {
+            margin-right: 5px;
+        }
+
+        input[type="button"] {
+            align-items: center;
+            background: #1785FB;
+            border-radius: 4px;
+            border: 0;
+            box-shadow: none;
+            color: #fff;
+            cursor: pointer;
+            display: inline-flex;
+            font: 500 12px/20px 'Roboto';
+            margin: 0;
+            padding: 6px 12px;
+            transition: all 0.2s ease;
+            vertical-align: top;
+        }
+
+        #map {
+            flex: 1;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="sidebar">
+        <h1>Torque</h1>
+        <h2>Controls</h2>
+        <div>
+            <input type="button" id="js-play-btn" value="PLAY">
+            <input type="button" id="js-stop-btn" value="STOP">
+            <input type="button" id="js-pause-btn" value="PAUSE">
+        </div>
+        <h2>Duration</h2>
+        <div>
+            <input type="range" id="js-duration-range" min="1" max="60" step="1">
+            <span id="js-duration-span"></span> 10 seconds
+        </div>
+
+        <h2>Time</h2>
+        <div>
+            <input type="range" id="js-time-range" min="0" max="1" step="0.01">
+        </div>
+    </div>
+
+    <div id="map"></div>
+    <script>
+        const map = new mapboxgl.Map({
+            container: 'map',
+            style: {
+                version: 8,
+                sources: {},
+                layers: [{
+                    id: 'background',
+                    type: 'background',
+                    paint: {
+                        'background-color': 'black'
+                    }
+                }]
+            },
+            center: [0, 0],
+            zoom: 0,
+            dragRotate: false
+        });
+
+        const animStart = '2018-04-11T12:00:00Z';
+        const animEnd = '2018-04-12T12:00:00Z';
+
+        const source = new carto.source.GeoJSON(generateData(), {
+            dateColumns: ['sim_time']
+        });
+
+        function generateData() {
```

Comment:
Can't we just link to one of our datasets on the Carto account?

Reply:
I prefer this approach for this kind of demo/test example, so it is self-contained (independent of any server data). @davidmanzanares you seem to think this kind of think doesn't fit in the examples folder... maybe we should have a demo/debug/xxx folder?

Reply:
(note that I've also added three similar animation demos in other branch)

Reply:
Yes, the examples folder is for documentation purposes to help end-users, we could have a `debugging` folder though. However, the functionality of having play/pause/stop buttons is useful for end-users, so, I think this example, or a similar one, should be added to `examples/`. Just with a better map.

Reply:
we can ask mamata and create a new issue ;)